app/views/articles/show.html.erb

```diff
@@ -120,25 +140,20 @@
             </div>
           </a>
         <% end %>
-        <h1 class="<%= @article.title_length_classification %>" itemprop="name headline">
-          <%= @article.title %>
-        </h1>
+        <h1 class="<%= @article.title_length_classification %>"></h1>
+
         <h3>
-          <span itemprop="author" itemscope itemtype="http://schema.org/Person">
-            <meta itemprop="url" content="<%= user_url(@user) %>">
-            <a href="/<%= @user.username %>" class="author">
-              <img class="profile-pic" src="<%= ProfileImage.new(@user).get(width: 50) %>" alt="<%= @user.username %> profile image" />
-              <span itemprop="name"><%= @user.name %></span>
```

Comment:
the span itself should stay in, otherwise we don't show the username anymore in the article page